src/main/scripts/travis-build.bash

```diff
@@ -79,7 +79,7 @@ function main() {
 
         # Build the Docker image only on releases
         if [[ $TRAVIS_TAG =~ ^[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
-            if ! $mvn docker:build -DpushImageTag; then
+            if ! $mvn docker:build -DpushImage; then
```

Comment:
Yeah, honestly, I find their README confusing here. I'm happy trying without the change.